### PR TITLE
[fontconfig] add vcpkg cmake wrapper.

### LINKED ISSUE #15490: [fontconfig] build failure
**Host Environment**
- OS: Linux Manjaro
- Compiler: g++

**To Reproduce**
`./vcpkg install fontconfig`

**Failure logs** 
```--- Configuring x64-linux-dbg
-- Configuring x64-linux-rel
-- Generating configure for x64-linux
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:108 (message):
    Command failed: /usr/bin/autoreconf -vfi
    Working Directory: /home/venny/projects/vctest/vcpkg/buildtrees/fontconfig/src/75c13b2421-4651fd65ce.clean/
    Error code: 1
    See logs for more information:
      /home/venny/projects/vctest/vcpkg/buildtrees/fontconfig/autoconf-x64-linux-out.log
      /home/venny/projects/vctest/vcpkg/buildtrees/fontconfig/autoconf-x64-linux-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_configure_make.cmake:536 (vcpkg_execute_required_process)
  ports/fontconfig/portfile.cmake:25 (vcpkg_configure_make)
  scripts/ports.cmake:136 (include)


Error: Building package fontconfig:x64-linux failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `./vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: fontconfig:x64-linux
  Vcpkg version: 2020.11.12-unknownhash
```

The mentioned log files are attached.
[autoconf-x64-linux-err.log](https://github.com/microsoft/vcpkg/files/5779601/autoconf-x64-linux-err.log)
[autoconf-x64-linux-out.log](https://github.com/microsoft/vcpkg/files/5779602/autoconf-x64-linux-out.log)

**Additional context**
I tried installing the nana package, but it failed at fontconfig.
Installing fontconfig directly renders the same result.

### PR DESCRIPTION
**Pull request by Neumann-A**
add a wrapper to fontconfig. Required for qt6 on linux

closes #15490 ?

### CODE REVIEW

**Comment by Neumann-A** [comment]
osg doing something stupid:
```
       (Link target) -> 
         LINK : fatal error LNK1104: cannot open file 'Freetype::Freetype.lib' [D:\buildtrees\osg\x64-windows-dbg\src\osgText\osgText.vcxproj]
```

**Comment by JackBoosY** [comment]
Looks good, is this PR ready for review?

**Comment by Neumann-A** [comment]
@JackBoosY: Yes I think so. The one corner case still open can be fixed if it is ever encountered.

File: ports/fontconfig/vcpkg-cmake-wrapper.cmake [hunk 1]
```diff
@@ -0,0 +1,46 @@
+_find_package(${ARGS})
```
Comment (strega-nil) [inline ports/fontconfig/vcpkg-cmake-wrapper.cmake:1]: This file should be called `vcpkg-cmake-wrapper.cmake.in`, but otherwise, this looks good to me.

**Comment by Hoikas** [comment]
CI Regressions are fixed in #15491

**Review by ras0219 — APPROVED** [review]

**Pull request merged**